packages/neo-one-website/docs/0-installation/0-quick-start.md

```diff
@@ -0,0 +1,285 @@
+
+```typescript
+/**
+ * @jest-environment node
+ */
+import { withContracts } from '../neo-one/test';
+
+describe('Token', () => {
+  test('exists', async () => {
+    // "token" is the contract name
+    await withContracts(async ({ token }) => {
+      expect(token).toBeDefined();
+    });
+  });
+});
+```
+
+::: warning
+
+Note
+
+`withContract()` is a generated helper that will expose all your contracts, their methods and properties along with other useful options. See [Testing](docs/testing) for more details.
+
+:::
+
+---
+
+## Quick Commands
+
+### Build
+
+```bash
+yarn neo-one build
+```
+
+Run `yarn neo-one build` when you are done making changes to the contract(s). It not only builds the necessary code for the contract(s) but also helps to update Intelisense for when you create tests. In details, the command will:
+
+1. Start up a local private NEO network (if one has not already been started).
+2. Setup wallets for testing with various amounts of NEO and GAS.
+3. Compile the project's smart contract(s).
+4. Deploy the smart contracts to the private Neo network.
+5. Generate code for use in your decentralized app.
+
+### Test
+
+Run `yarn test` to run your tests.
+
+```bash
+yarn test
+```
+
+::: warning
+
+Note
+
+For more command line help and options, check out the [CLI](/docs/cli) page.
+
+:::
+
+---
+
+## Client Integration
+
+Integrating the NEO•ONE client APIs in a vanilla JavaScript or TypeScript application is very simple - assuming we have a contract called `Token` and we’re in the `src/index.ts` file using the default NEO•ONE toolchain paths:
+
+```typescript
+import { createClient, createTokenSmartContract } from './neo-one';
+
+const client = createClient();
+const token = createTokenSmartContract(client);
+// "token" is the name of this example smart contract
+// for example:
+// if you contract name is Test
+// then the function will become createTestSmartContract()
+```
+
+Check out our [Client APIs](/docs/client-apis) page for more details.
+
+---
+
+## Deploy
+
+::: warning
+
+Important
+
+If you do not know what contracts need to be deployed, check the following
+[What Contracts Need To Be Deployed?](https://docs.neo.org/docs/en-us/sc/deploy/deploy.html#what-contracts-need-to-be-deployed)
+
+:::
+
+### Checklist:
+
+- Wallet's private key
+- Test coins
+- Migration file
+
+You first need a wallet and the wallet's private key and sufficient amount of NEO and GAS to deploy.
+
+### Wallet set-up:
+
+1. Go to `https://neotracker.io/`
+2. Go to `“Wallet”` tab.
+3. Click on `“New Wallet”`.
+4. Enter a password (make sure you save this password), which is used to unlock the wallet later.
+5. Click on `"Download Encrypted Key"` (this will generate a `.txt` file that can be used to unlock your wallet via `“Keystore File”` option).
+6. Save the private key shown on screen locally and secure it.
+7. Click continue to view your wallet.
+8. (Optional) Generate a PDF by clicking on “Print paper wallet”
```

Comment:
"...a PDF of your wallet..."

Reply:
Surround "Print paper wallet" in backticks like you do everywhere else: `"Print paper wallet"`.